certify-sunbird-insurance.properties

```diff
@@ -102,10 +104,10 @@ mosip.certify.key-values={\
    'vd12' : {\
               'credential_issuer': '${mosip.certify.identifier}',   \
               'authorization_servers': {'${mosip.certify.authorization.url}'}, \
-              'credential_endpoint': '${mosipbox.public.url}${server.servlet.path}/issuance/vd12/credential', \
+              'credential_endpoint': '${mosip.certify.domain.url}${server.servlet.path}/issuance/vd12/credential', \
```

Comment:
I think this and identifier should also be overrided here to allow the different URLs

Reply:
added identifier and domain.url as overrided properties in all plugin property files